### ISSUE TITLE
Clean up workbench_utils.py

### ISSUE THREAD

**Issue by mjordan** [mjordan/islandora_workbench#87]
Once the issue-56 branch is merged into master, clean up workbench_utils.py:

* remove `import collections` and `import pprint / pp = pprint.PrettyPrinter(indent=4)`
* put back `@lru_cache(maxsize=None)`
* add an inline comment explaining why we `.remove()` entries from `csv_column_headers` (it's so the subsequent `if csv_column_header not in drupal_fieldnames` check will not complain).

**Comment by mjordan** [comment]
Also did a bit of cleanup in CreatePagedContentTest.py.
